core/spec/helpers/base_helper_spec.rb

```diff
@@ -134,4 +134,71 @@ def link_to_tracking_html(options = {})
       expect(pretty_time(DateTime.new(2012, 5, 6, 13, 33))).to eq "May 06, 2012  1:33 PM"
     end
   end
+
+  describe '#display_price' do
+    let!(:product) { create(:product) }
+    let(:current_currency) { "USD" }
+
+    context "when there is no current order" do
+      let (:current_tax_zone) { nil }
+
+      it 'returns the price including default vat' do
+        expect(display_price(product)).to eq('$19.99')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.